test_elasticsearch_dsl/test_aggs.py

```diff
@@ -166,3 +166,7 @@ def test_filters_correctly_identifies_the_hash():
         }
     } == a.to_dict()
     assert a.filters.group_a == query.Q('term', group='a')
+
+def test_import__all__():
+    for name in aggs.__all__:
+        exec('from elasticsearch_dsl.aggs import {0}'.format(name))
```

Comment:
I think `assert hasattr(aggs, name)` would be better here, `exec` is scary even in tests :)